### PR TITLE
Story1

### PR DESCRIPTION
**Pull request by FrankieD96**

### CODE REVIEW

File: src/home.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<?php
```
Comment (ashleycoles) [inline src/home.php:1]: I don't think you need this file any more

File: ShirtClass.php [hunk 2]
```diff
@@ -0,0 +1,45 @@
+<?php
```
Comment (ashleycoles) [inline ShirtClass.php:1]: Extra file?

File: ShirtsModel.php [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+            echo "<p>" . $shirt['season'] . " " . $shirt['type'] . " " . "kit</p>";
+            echo "<div>";
+            echo "<ul>";
+            echo "<li>{$shirt['league_name']}";
+            echo "<li>{$shirt['country']}";
+            echo "<li>Made by {$shirt['brand_name']}";
+            echo "</div>";
+            echo "<img class='image' src='{$shirt['img_url']}'>";
+            echo "</div>";
+        }
```
Comment (ashleycoles) [inline ShirtsModel.php]: The job of a model is never anything beyond interacting with a database, getting info, inserting info, etc. So displaying in here violates the single responsibility principle. This method should purely return $shirts, not do anything with it. It should be displayed elsewhere

Comment (ashleycoles) [reply ShirtsModel.php]: This is also where you need to be looking at creating and using a Shirt entity class. Did you look at the example in that pdf or manage to find the products example we did in clas?

File: ShirtsModel.php [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+            INNER JOIN `brands`
+            ON `shirts`.`brand_id` = `brands`.`id`
+            INNER JOIN `leagues`
+            ON `teams`.`league_id` = `leagues`.`id`
+            INNER JOIN `countries`
+            ON `leagues`.`country_id` = `countries`.`id`;");
+        
+        $query->execute();
+        
+        $shirts = $query->fetchALL();
```
Comment (ashleycoles) [inline ShirtsModel.php]: It should be fetchAll without the big Ls

File: index.php [hunk 5]
```diff
@@ -18,10 +18,33 @@
...
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+?>
 
+<body>
+    <nav class = 'navbar'>
+        <h2>Footie Shirts: My Collection</h2>
+        <div class = 'nav-links'>
+            <a href="index.php">Home</a>
+            <a href="#">Add</a>
+            <a href="#">Edit</a>
```
Comment (ashleycoles) [inline index.php]: You should remove these two links for now, you never want to have code in that doesn't fully work

File: index.php [hunk 6]
```diff
@@ -18,10 +18,33 @@
...
+        </div>  
+        <form>
+            <input type="text" placeholder="search" name="search">
+            <button type=submit>Search</button>  
+        </form>
```
Comment (ashleycoles) [inline index.php]: Same with this

**Review by ashleycoles — CHANGES_REQUESTED** [review]

File: ShirtsModel.php (hunk 3 above)
Comment (FrankieD96) [reply ShirtsModel.php]: I got a little confused and just submitted what I had managed, I shall look at sorting this now

File: ShirtsModel.php (hunk 4 above)
Comment (FrankieD96) [reply ShirtsModel.php]: done

File: index.php (hunk 5 above)
Comment (FrankieD96) [reply index.php]: done

File: index.php (hunk 6 above)
Comment (FrankieD96) [reply index.php]: done

**Review by ashleycoles — APPROVED** [review]

**Pull request merged**